posthog/api/person.py

```diff
@@ -101,6 +103,26 @@ class Meta:
             "created_at",
             "uuid",
         ]
+        read_only_fields = ("id", "distinct_ids", "created_at", "uuid")
+
+    def update(self, instance: Person, validated_data: Dict):
+        # :KLUDGE: Keep clickhouse in sync with any property updates.
+        # Note we don't update the person model here as plugin-server is responsible for that.
+        capture_internal(
+            distinct_id=instance.distinct_ids[0],
+            ip=None,
+            site_url=None,
+            team_id=instance.team_id,
+            now=datetime.now(),
+            sent_at=None,
+            event={
+                "event": "$set",
+                "properties": {"$set": validated_data["properties"]},
+                "distinct_id": instance.distinct_ids[0],
+                "timestamp": datetime.now().isoformat(),
+            },
+        )
```

Comment:
should we `log_activity` here?

not too familiar with this whole thing but yeah

Reply:
This seems to be already handled according to tests.